src/main/java/essenmakanan/command/AddIngredientCommand.java

```diff
@@ -5,18 +5,20 @@
 import essenmakanan.recipe.RecipeList;
 
 public class AddIngredientCommand extends Command{
+    private String toAdd;
 
-    public AddIngredientCommand() {
+    public AddIngredientCommand(String toAdd) {
         super(false);
+        this.toAdd = toAdd;
     }
 
     public String parseIngredientTitle(String input) {
         return input.replace("i/", "");
```

Comment:
is this still necessary? i notice you removed this part from the AddRecipeCommand as well